### DISCUSSION TITLE
keyboard popup layout

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10654] [Q&A] [answered]
Is there a way to leave the top left key empty on the additional upper row of a keyboard popup and just use the middle key?

Thanks!

**Comment by Frenzie** [comment]
What have you tried that didn't work?

**Reply by ichnilatis-gr** [reply]
I wrote in the last lines:

    "",
    { label = "◌͂", key = "͂", }, -- Combining Greek Perispomeni

but this creates a white empty key, not a grey empty place.

**Reply by ichnilatis-gr** [reply]
Well, it's just for appearance and a little bit for practical reasons... :-)

**Reply by Frenzie** [reply]
Oh right, those aren't positional, just extra.

**Reply by ichnilatis-gr** [reply]
So, there is no way. Right?

**Reply by Frenzie** [reply]
I don't think so, no.

**Comment by poire-z** [comment, marked as answer]
Looks like using `false` instead of a string or table works (by luck may be :) - if I understand right what you want:

![image](https://github.com/koreader/koreader/assets/24273478/43572062-d3a7-4554-8b68-6d1ab0b25f12)
```diff
--- a/frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua
+++ b/frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua
@@ -44,7 +44,7 @@ return {
         south = "|",
         southeast = "\\",
         southwest = "/",
-        '^',
+        false,
         "&",
         "$",
     },
```

**Reply by ichnilatis-gr** [reply]
That's what I wanted. Thank you. But it looks like I won't have to use it after all. I'll see. I'm working on it...
Anyway, thank you again. It may be useful.

**Comment by Frenzie** [comment]
It's probably by design, just not by design for there specifically.
